src/pymatgen/io/vasp/outputs.py

```diff
@@ -3568,14 +3568,14 @@ def parse_file(filename: PathLike) -> tuple[Poscar, dict, dict]:
 
     def write_file(
         self,
-        file_name: str | Path,
+        file_name: PathLike,
         vasp4_compatible: bool = False,
     ) -> None:
         """Write the VolumetricData object to a VASP compatible file.
 
         Args:
-            file_name (str): Path to a file
-            vasp4_compatible (bool): True if the format is VASP4 compatible
+            file_name (PathLike): The output file.
+            vasp4_compatible (bool): Whether the format is VASP4 compatible.
```

Comment:
i actually prefer the `True if` phrasing here since it leaves no ambiguity on the boolean's direction

Reply:
I think both should be clear enough, changed this because I thought in previous cleanups the `whether` phrasing is used (perhaps just me 🙈 ), perhaps would just leave it as is in the future as they're pretty much interchangeable.